smart-workflow/src/com/axonivy/utils/smart/workflow/model/ChatModelFactory.java

```diff
@@ -16,16 +18,19 @@
 
 public class ChatModelFactory {
 
+  private static final String FALLBACK_PROVIDER = "OpenAI";
+  private static final String SMART_WORKFLOW_PROJECT_PREFIX = "smart-workflow";
```

Comment:
This string here is to identify the current PMV of this project. It hits only one and is not meant as prefix here. So I'd omit `_PREFIX` from the variable name.